src/test/rpc_wallet_tests.cpp

```diff
@@ -626,7 +626,7 @@ BOOST_AUTO_TEST_CASE(rpc_wallet_z_importexport)
     BOOST_CHECK(myaddrs == listaddrs);
 
     // Add one more address
-    BOOST_CHECK_NO_THROW(retValue = CallRPC("z_getnewaddress"));
+    BOOST_CHECK_NO_THROW(retValue = CallRPC("z_getnewaddress sprout"));
     std::string newaddress = retValue.get_str();
     auto address = DecodePaymentAddress(newaddress);
     BOOST_CHECK(IsValidPaymentAddress(address));
```

Comment:
github doesn't let me comment on lines outside the narrow range of lines that include diffs, but I happened to notice these lines (starting at line 637) could be improved:
```
    // Check if too many args
    BOOST_CHECK_THROW(CallRPC("z_getnewaddress toomanyargs"), runtime_error);
```

Reply:
I notice that's the new idiom that is being used in some of the tests.  Please file a ticket and we can improve at a future date.

Reply:
Done, #3687. I'm not sure I understand what you mean about a new idiom; please update that issue if you think it's lacking.